### DISCUSSION TITLE
Enhance Github security by connecting using Secure Shell Protocol (SSH) | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#75] [Announcements]
# Enhance Github security by connecting using Secure Shell Protocol (SSH) | Leo's dev blog – stories, insights, and ideas

Using the SSH protocol, you can connect and authenticate to remote Github servers and services without supplying your username and personal access token at each visit.

https://www.leohuynh.dev/snippets/connecting-to-github-using-ssh

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
